src/js/Content/Features/Common/FEarlyAccess.ts

```diff
@@ -150,8 +150,7 @@ export default class FEarlyAccess extends Feature {
     // TODO support React-based sales pages, curator lists, etc.
     static _selector = [
```

Comment:
Is that even used anywhere?

Reply:
I thought it had something to do with the ongoing refactor

Reply:
We'll see that then I guess :D